x-pack/plugins/apm/server/lib/alerts/register_transaction_duration_alert_type.ts

```diff
@@ -7,21 +7,21 @@
 import { schema } from '@kbn/config-schema';
 import { Observable } from 'rxjs';
 import { take } from 'rxjs/operators';
-import { getDurationFormatter } from '../../../common/utils/formatters';
-import { ProcessorEvent } from '../../../common/processor_event';
```

Comment:
Didn't we use to have an eslint rule that'd strictly order the imports alphabetically? Then we'd avoid this kind of diff.

I use that for backport: https://github.com/sqren/backport/blob/9c41c5bcd559f1b1b7a782a940143d28d1027c79/.eslintrc.js#L25

Reply:
I don't know, but once they are ordered properly this kind of diff doesn't need to happen again.

Reply:
The ordering I'm doing here is with VSCode's options-shift-o, which invokes TypeScript's [organize imports](https://devblogs.microsoft.com/typescript/announcing-typescript-2-8-2/#organize-imports) feature.

We have some ordering rules in ESLint, but it doesn't fully check for TypeScript's suggested ordering. There's an open issue (typescript-eslint/typescript-eslint#113) to get it implemented in the ESLint TypeScript plugin that's not yet completed.

Reply:
FWIW, it's a little harder to review this PR because of the import changes. I would prefer to have something that is enforced & automatic, or to not do this at all. I do agree that there is value in sorting the imports.

Reply:
Agree, there is value in sorting imports and it should be enforced and automatic. For backport unsorted imports make the eslint check fail:

![image](https://user-images.githubusercontent.com/209966/98798760-88e84e80-240e-11eb-958a-c1ea2fa738bd.png)

![image](https://user-images.githubusercontent.com/209966/98798749-8685f480-240e-11eb-92cf-b5d52460e847.png)